src/account/AccountItem.js

```diff
@@ -26,34 +26,54 @@ const styles = createStyleSheet({
     marginLeft: 16,
   },
   text: {
-    color: BRAND_COLOR,
     fontWeight: 'bold',
     marginVertical: 2,
   },
   icon: {
     padding: 12,
     margin: 12,
   },
+  signOutText: {
+    fontStyle: 'italic',
+    fontSize: 12,
+    textAlign: 'right',
+    position: 'absolute',
+    end: 10,
+    top: 3,
+    color: 'rgba(0,0,0,.5)',
+  },
 });
 
 type Props = $ReadOnly<{|
-  index: number,
-  email: string,
-  realm: URL,
+  accounts: AccountStatus,
   onSelect: (index: number) => void,
   onRemove: (index: number) => void,
-  showDoneIcon: boolean,
+  index: number,
 |}>;
 
 export default function AccountItem(props: Props) {
-  const { email, realm, showDoneIcon } = props;
+  const { email, realm, isLoggedIn } = props.accounts;
+  const showDoneIcon = Boolean(props.index === 0 && isLoggedIn);
+  const backgroundItemColor = isLoggedIn ? 'hsla(177, 70%, 47%, 0.1)' : 'rgba(0,0,0,.07)';
+  const textColor = isLoggedIn ? BRAND_COLOR : 'rgba(0,0,0,.6)';
 
   return (
     <Touchable style={styles.wrapper} onPress={() => props.onSelect(props.index)}>
-      <View style={[styles.accountItem, showDoneIcon && styles.selectedAccountItem]}>
+      <View
+        style={[
+          styles.accountItem,
+          showDoneIcon && styles.selectedAccountItem,
+          { backgroundColor: backgroundItemColor },
+        ]}
+      >
+        {!isLoggedIn && <Text style={styles.signOutText}>Signed out</Text>}
```

Comment:
Let's use `Label` instead of `Text`, and add the new user-facing string to static/translations/messages_en.json, to wire it up for [translation](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/translations.md).